handbook/engineering/index.md

```diff
@@ -16,6 +16,7 @@
 - [Continuous releasability](continuous_releasability.md)
 - [Commit message guidelines](commit_messages.md)
 - [Ignoring editor config files in Git](ignoring_editor_config_files.md)
+- [Configuring Zoom to send recordings to Slack automatically](configuring_zoom_recordings_to_slack_automatically.md)
```

Comment:
I remember some discussion in slack a while back about disabling zoom auto-recording - maybe zoom tips in general warrants its own page? ie `zoom.md` -> section `zoom recordings ...`

Reply:
That would be great to have indeed!